src/pages/Account.vue

```diff
@@ -7,38 +7,43 @@
 
                 <transition name="fade"><div v-if="errorMessage" v-bind:class="[goodFeedback ? 'good message' : 'message' ]">{{ errorMessage }}</div></transition>
 
+                <div v-if="voucherPayments.length > 0">
```

Comment:
This check will have to be changed a little because voucherPayments.length always returns 1. It's because Vue is attaching an Observer I believe.